alpine-base-s6/Makefile

```diff
@@ -1,5 +1,5 @@
-include ../../Makefile.conf
 include ../Makefile.conf
+include ./Makefile.conf
```

Comment:
I'd remove Makefile.conf and just specify the IMAGE= in Makefile, since it's not shared.